nevergrad/optimization/oneshot.py

```diff
@@ -65,18 +89,21 @@ class RandomSearchMaker(base.ParametrizedFamily):
 
     # pylint: disable=unused-argument
     def __init__(self, *, middle_point: bool = False, stupid: bool = False,
+                 quasi_opposite: str = "none",
                  cauchy: bool = False, scale: Union[float, str] = 1.) -> None:
         # keep all parameters and set initialize superclass for print
         assert isinstance(scale, (int, float)) or scale == "random"
         self.middle_point = middle_point
```

Comment:
Add:
`assert quasi_opposite is None or quasi_opposite in ["quasi", "opposite"]`

Reply:
Done.